src/index.html

```diff
@@ -1,13 +1,496 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>WITCH OF MOUNTAIN WINDS</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      rel="shortcut icon" 
+      href="./images/svg/logo-colour.svg" 
+      type="image/png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600&display=swap" rel="stylesheet">
```

Comment:
The given fonts are not the ones requested in the task description. You should use Sharp Sans No2 Medium and Avenir fonts instead of Poppins.